tests/dragonfly/redis_replication_test.py

```diff
@@ -316,3 +316,4 @@ async def run_replication(c_replica):
     await check_data(seeder, replicas, c_replicas)
 
     await proxy.close(proxy_task)
+    await asyncio.sleep(0.1)
```

Comment:
I thought the issue was that we did not yield to the main loop but I was wrong. Will address this soon

Reply:
I could not reproduce the warning locally and is this was a side fix I don't want to loose time on this